Sources/Extensions/SpriteKit/SKPhysicsBodyExtensions.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SKPhysicsBodyExtensions.swift
+//  SwifterSwift
+//
+//  Created by Robbie Moyer on 7/20/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(SpriteKit)
+import SpriteKit
+
+// MARK: - Methods
+public extension SKPhysicsBody {
+    
+    /// SwifterSwift: Checks if a physics body is contacting another body
+    /// with the given `categoryBitMask`
+    ///
+    ///     if player.physicsBody?.isTouching(Category.fire) {
+    ///         // Reduce player health
+    ///     }
+    ///
+    /// - Parameter category: The `categoryBitmMask` to check
+    /// - Returns: `true` if the physics body is contacting another body
+    ///             with the given bitmask, `false` otherwise.
+    public func isTouching(_ category: UInt32) -> Bool {
+        return allContactedBodies().contains(where: { $0.categoryBitMask == category })
```

Comment:
This is incorrect. To check that a bitmask intersects you need use a logical AND:
`($0.categoryBitMask & category) != 0`.